services/crudService.js

```diff
@@ -14,6 +14,13 @@ const findById = (model, res, id, isDataSend = true) => {
         .catch(err => sendError(res, err.message, err.code));
 };
 
+const findByKey = (model, res, params, isDataSend = true) => {
+    return model
+        .findOne(params)
+        .then(data => isDataSend ? res.status(200).send(data) : data)
```

Comment:
I`ve seen `res.status(200).send` somewhere, and `res.status(200).json` somewhere else.
You better create some wrapper sendSuccess(data)

Reply:
Yeah. Good idea